### PR TITLE
Company entity type

### PR DESCRIPTION
**Pull request by kiproping**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem
Company entity type model was missing

## How I resolved it
Added company entity type

## How to verify it

testing

## Questions

None

## Notes

...

Notify the following people: @darrynten

### CODE REVIEW

File: tests/mocks/CompanyEntityType/GET_CompanyEntityType_Get.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      "CountryId": 2,
+      "Name": "sample string 3",
+      "OwnershipDescription": "sample string 4",
+      "HoldingDescription": "sample string 5",
+      "HoldingUnit": "sample string 6",
+      "Created": "2017-07-19",
+      "CanIssueShares": true
+    },
+    {
+      "ID": 1,
```
Comment (fergusdixon) [inline tests/mocks/CompanyEntityType/GET_CompanyEntityType_Get.json]: Results[0] and results[1] should have unique data.

**Review by fergusdixon — CHANGES_REQUESTED** [review]
Mocks need to be updated

**Review by fergusdixon — DISMISSED** [review]

File: tests/SageOne/Models/CompanyEntityTypeModelTest.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            ],
+        ]);
+    }
+
+    public function testGetId()
+    {
+        $this->verifyGetId(CompanyEntityType::class, 1, function ($model) {
+            $this->assertEquals(1, $model->id);
+            $this->assertEquals(2, $model->countryId);
+            $this->assertEquals("sample string 3", $model->name);
```
Comment (darrynten) [inline tests/SageOne/Models/CompanyEntityTypeModelTest.php]: These should be single quotes

**Review by darrynten — CHANGES_REQUESTED** [review]
There's some double quotes that should be single quotes but other than that looking great!

**Review by darrynten — APPROVED** [review]
Nice one! :D

**Pull request merged**